openedx/core/djangoapps/credit/serializers.py

```diff
@@ -29,6 +29,24 @@ class CreditCourseSerializer(serializers.ModelSerializer):
 
     course_key = CourseKeyField()
 
-    class Meta(object):
+    class Meta(object):  # pylint: disable=missing-docstring
         model = CreditCourse
         exclude = ('id',)
+
+
+class CreditProviderSerializer(serializers.ModelSerializer):
+    """ CreditProvider Serializer """
+
+    class Meta(object):  # pylint: disable=missing-docstring
+        model = CreditProvider
+        exclude = ('id', 'eligibility_email_message', 'receipt_email_message')
```

Comment:
Use `fields` instead. If we ever add fields to the model, we don't want to expose them accidentally.